docker-compose.services.yml

```diff
@@ -70,10 +70,13 @@ services:
 
   matomo:
     build:
-      context: ../analytics/
+      # Uncomment the below line and comment out the line below that if you want to use the local Dockerfile
+      # context: ../analytics/
+      context: https://github.com/USSF-ORBIT/analytics#main
       dockerfile: Dockerfile
     ports:
       - '8443:8443'
+      - '8081:8080'
```

Comment:
Question: Should we update `.envrc` to point to `http://localhost:8081` for matomo as well?

Reply:
That's a good point. I was going to tell people to edit their `.envrc.local` but there's not reason to not update the `.envrc` file.